db/migrate/20200726022620_add_stripe_customer_id_to_users.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class AddStripeCustomerIdToUsers < SolidusSupport::Migration[5.1]
+  def up
+    add_column :spree_users, :stripe_customer_id, :string, unique: true
```

Comment:
There's a concern regarding this migration. On [custom authentications](https://guides.solidus.io/developers/users/custom-authentication.html), the table may not be `:spree_users` (or Spree::User model). This migration would need to find the correct user class through `Spree.user_class`.

Reply:
Good point! I changed `:spree_users` to `Spree.user_class.table_name.to_sym`.